app/src/main/java/fr/free/nrw/commons/upload/UploadClient.kt

```diff
@@ -80,15 +78,23 @@ class UploadClient @Inject constructor(
         compositeDisposable.add(
             Observable.fromIterable(fileChunks).forEach { chunkFile: File ->
                 if (canProcess(contribution, failures)) {
-                    processChunk(
-                        filename, contribution, notificationUpdater, chunkFile,
-                        failures, chunkInfo, index, errorMessage, mediaType!!, file!!, fileChunks.size
-                    )
+                    if (CommonsApplication.cancelledUploads.contains(contribution.pageId)) {
+                        compositeDisposable.clear()
+                        return@forEach
+                    }else{
```

Comment:
Unfortunately, we still don't have a linter on our repository. Would you mind making the spaces around if-else blocks more consistent? We follow the [Java style guide](https://google.github.io/styleguide/javaguide.html).